flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/source/FileStoreSourceSplitReaderTest.java

```diff
@@ -64,32 +67,92 @@ public static void after() {
     }
 
     @Test
-    public void testKeyAsRecord() throws Exception {
-        innerTestOnce(true);
+    public void testPrimaryKey() throws Exception {
+        innerTestOnce(false, 0);
+    }
+
+    @Test
+    public void testValueCount() throws Exception {
+        innerTestOnce(true, 0);
```

Comment:
Correct me if I'm wrong. `innerTestOnce` relies on `TestDataReadWrite`. But the latter hardcoded the key type, value type, and accumulator type to the pk situation.

Reply:
value is a bigint.
- has pk, value is the record
- value count, value is the record count

Reply:
> value is a bigint.
> 
> * has pk, value is the record
> * value count, value is the record count

Got that. when the record has no pk, the kV pairs stands for <record, occurrence>, and the sign stands for add/delete